src/tests/util.ts

```diff
@@ -0,0 +1,23 @@
+import { algorandFixture } from '@algorandfoundation/algokit-utils/testing'
+import { microAlgos } from '@algorandfoundation/algokit-utils'
+
+export const wait = (ms: number): Promise<void> => new Promise((resolve) => setTimeout(() => resolve(), ms))
+
+export const setUpLocalnet = async () => {
+  const localnet = algorandFixture({
+    testAccountFunding: microAlgos(100_000_000_000),
+  })
+
+  if (process.env.TEST_ENV == 'ci') {
+    /*
+     algokit localnet start is currently not waiting for the indexer to start which is leading to test failures, so we'll wait for it here
+     until that's patched.
+
+     This wait is not required locally as localnet has likely already been started for some time.
+   */
+
+    await wait(1000)
```

Comment:
I wonder if instead we wait every 100ms for the indexer port to be open? It'll be more reliable.

Reply:
hmm, I'm hoping this is temporary 😝